draft-yasskin-http-origin-signed-responses.md

```diff
@@ -331,7 +331,7 @@ Accept: */*
 
 HTTP/1.1 200
 Content-Type: text/html
-MI: mi-sha256=20addcf7368837f616d549f035bf6784ea6d4bf4817a3736cd2fc7a763897fe3
+MI: mi-sha256=dcRDgR2GM35DluAV13PzgnG6-pvQwPywfFvAu1UeFrs==
```

Comment:
I was wondering if this should be
```
MI: mi-sha256=*dcRDgR2GM35DluAV13PzgnG6-pvQwPywfFvAu1UeFrs==*
```

but it looks like https://martinthomson.github.io/http-mice/draft-thomson-http-mice.html#header refers to RFC7230 and RFC7231, not https://tools.ietf.org/html/draft-ietf-httpbis-header-structure-06 yet.

Reply:
@nyaxt right, MICE doesn't use structured headers.

OT: where can I ask questions/discuss on other parts of this proposal?

Reply:
OT: Filing issues here is great. :)